### PR TITLE
Feature/create restaurant subscription page

### PR DESCRIPTION
**Pull request by mzelens**
Draft of the restaurant subscription / registration page. No responsiveness yet.

### CODE REVIEW

File: src/features/client/pages/Subscriptions/index.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import RestaurantPromo2 from '../../components/RestaurantSignUp/RestaurantPromo2';
+import SubscriptionPlans from '../../components/RestaurantSignUp/SubscriptionPlans';
+import SubmitForm from '../../components/RestaurantSignUp/SubmitForm';
+import { ChakraProvider } from '@chakra-ui/react';
+import './index.scss';
+
+function Subscriptions() {
+  return (
+    <ChakraProvider>
+      <RestaurantPromo1 />
```
Comment (dmiknys) [inline src/features/client/pages/Subscriptions/index.jsx:13]: Dont use numbers in components names. Try naming it differently

File: src/features/client/components/RestaurantSignUp/RestaurantPromo1/index.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            <h4 className="promo1__list-item">
+              Signup discount of <span>50%</span>
+            </h4>
+            <h4 className="promo1__list-item">
+              From <span>12%</span> commission
+            </h4>
+          </ul>
+          <div className="promo1__btns">
+            {/* <Button content="Sign up for bfd.lt" isSubmit={false} />
+            <Button content="Sign up for a free workshop" isSubmit={false} /> */}
```
Comment (dmiknys) [inline src/features/client/components/RestaurantSignUp/RestaurantPromo1/index.jsx:26]: dont leave comments pls

**Review by dmiknys — CHANGES_REQUESTED** [review]

File: src/features/client/pages/Subscriptions/index.jsx (hunk 1 above)
Comment (fullofheal) [reply src/features/client/pages/Subscriptions/index.jsx]: fixed

File: src/features/client/components/RestaurantSignUp/SubmitForm/Thanks.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        size="lg"
+        onClick={() => hideThanks()}
+      >
+        Back
+      </Button>
+    </div>
+  );
+}
+
+export default Thanks;
```
[the remark this answers is not in the document]
Comment (fullofheal) [reply src/features/client/components/RestaurantSignUp/SubmitForm/Thanks.jsx]: added chakra modal

File: src/features/client/components/RestaurantSignUp/RestaurantPromo1/index.jsx (hunk 2 above)
Comment (fullofheal) [reply src/features/client/components/RestaurantSignUp/RestaurantPromo1/index.jsx]: cleaned comments

File: src/features/client/components/RestaurantSignUp/SubmitForm/index.jsx [hunk 4]
```diff
@@ -39,16 +55,11 @@ function SubmitForm() {
...
   };
 
-  const hideThanks = () => {
-    setThanks(false);
-    document.body.style.overflow = 'unset';
-  };
-
   const onSubmit = (data) => {
     if (checkBoxes.webshopChecked || checkBoxes.inAppChecked) {
       console.log(data);
```
Comment (dmiknys) [inline src/features/client/components/RestaurantSignUp/SubmitForm/index.jsx:60]: dont leave debug logs pls

**Review by dmiknys — APPROVED** [review]

File: src/features/client/components/RestaurantSignUp/SubmitForm/index.jsx (hunk 4 above)
Comment (fullofheal) [reply src/features/client/components/RestaurantSignUp/SubmitForm/index.jsx:60]: good point. missed that.

**Pull request merged**